CONTRIBUTING.md

```diff
@@ -63,7 +63,8 @@ you will most likely use this for WSL), `homebrew` (macOS / Linux), and more:
 - A Java 8 or later JDK (packages vary, use Google/DuckDuckGo/etc.).  
 If you need one, you can find them on [AdoptOpenJDK](https://adoptopenjdk.net/).
 - `maven` (often package `maven`; can be found on
-[Apache's site](https://maven.apache.org/download.cgi) too).
+[Apache's site](https://maven.apache.org/download.cgi) too);
```

Comment:
Why the semicolon?

Reply:
nvm, guess it's consistent with the ones above it

Reply:
Git and Patch both have semi colons at the end, showing they aren't the end of the requirements. Java, for whatever reason, does not have a semi-colon.